getting-started/setup-multi-node-basic.md

```diff
@@ -0,0 +1,110 @@
+# Set up multi-node TimescaleDB [](basic-multi-node-setup)
+
+A multi-node TimescaleDB implementation consists of: 
+- One access node to handle ingest, data routing and act as an entry 
+point for user access; 
+- One or more data nodes to store and organize distributed data.     
+
+All nodes begin as single-node TimescaleDBs, i.e., hosts with a running 
+PostgreSQL instance and a loaded TimescaleDB extension.  More detail on 
+the architecture can be found [here][architecture]. The steps for 
+creating a basic multi-node setup are as follows:
+
+## Basic setup
+1. Configure data nodes for communication with the access node
+1. Initialize data nodes from the access node 
+
+### Prerequisites
+- One PostgreSQL instance to act as an access node
+- One or more PostgreSQL instances to act as data nodes
+- TimescaleDB [installed][install] and [set up][setup] on all instances
+- Access to a superuser role (e.g. `postgres`) on all instances
+
+### 1. Configure data nodes for node-to-node communication
+To enable communication between the access node and the data node(s), the 
+underlying PostgreSQL instances for each data node must be configured to 
+authorize external communication.  The simplest way is to use "trust" 
+authentication which provides unencumbered client access to the instances.
```

Comment:
I would avoid saying "external communication" here as it sounds as if one must accept connections from outside the internal network.

```suggestion
authorize user connections from the access node.  The simplest way is to use "trust" 
authentication which allows the access node to connect and start sessions on data nodes on behalf of any user in the database.
```